contracts/curation/Curation.sol

```diff
@@ -2,17 +2,20 @@
 
 pragma solidity ^0.7.6;
 
-import "@openzeppelin/contracts/utils/Address.sol";
-import "@openzeppelin/contracts/math/SafeMath.sol";
-import "@openzeppelin/contracts/proxy/Clones.sol";
-
-import "../bancor/BancorFormula.sol";
-import "../upgrades/GraphUpgradeable.sol";
-import "../utils/TokenUtils.sol";
-
-import "./CurationStorage.sol";
-import "./ICuration.sol";
-import "./GraphCurationToken.sol";
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
```

Comment:
Ah, because when I updated Managed to use this format, it turned out Curation was relying on "accidentally" importing stuff from there, so I had to change it here for the build to succeed again